src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Colors.values()[new Random().nextInt(Colors.values().length)].toString();
```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method. Also lets use `name()` method instead of `toString()` here

Reply:
Did not understand about name() method